src/OpenTelemetry.Exporter.Prometheus.AspNetCore/OpenTelemetry.Exporter.Prometheus.AspNetCore.csproj

```diff
@@ -0,0 +1,42 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <!-- OmniSharp/VS Code requires TargetFrameworks to be in descending order for IntelliSense and analysis. -->
+    <TargetFrameworks>netcoreapp3.1</TargetFrameworks>
+    <Description>Prometheus exporter for AspNetCore for OpenTelemetry .NET</Description>
```

Comment:
nit: I feel like we could be a little more descriptive here. How about...

`AspNetCore middleware for hosting OpenTelemetry .NET Prometheus exporter`